java/springboot-hikari-ucp/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.0.0</version>
+		<relativePath /> <!-- lookup parent from repository -->
+	</parent>
+
+	<groupId>com.oracle.dev.jdbc</groupId>
+	<artifactId>springboot-hikari-ucp</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<name>springboot-hikari-ucp</name>
+	<description>A Guide to Migrating from HikariCP to Oracle UCP (Universal
+		Connection Pool)</description>
+	<url>https://juarezjunior.medium.com/e2f3fb6db2d5</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>17</maven.compiler.source>
+		<maven.compiler.target>17</maven.compiler.target>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ojdbc11</artifactId>
+			<version>23.3.0.23.09</version>
+		</dependency>
+		<dependency>
+			<groupId>com.oracle.database.jdbc</groupId>
+			<artifactId>ucp11</artifactId>
+			<version>23.3.0.23.09</version>
```

Comment:
Please use 23.4

Reply:
Done

Reply:
Please use [23.4.0.24.05](https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc11/23.4.0.24.05)

Reply:
Hi Jean, it's already using it. You're looking at a past commit, I followed all your requests. To confirm, look at commit 75a6f82896adb503af24d42cf1e149cb96689c02

Reply:
@Kuassim please review, thanks